config/jobs/kubernetes/sig-k8s-infra/periodics.yaml

```diff
@@ -200,6 +200,7 @@ periodics:
           - --
           - --check-leaked-resources
           - --env=KUBE_FEATURE_GATES=DisableCloudProviders=false,DisableKubeletCloudCredentialProviders=false
```

Comment:
don't we want these to be `true` going forward?

Reply:
we have to audit after we dig out from under